collector/pkg/component/consumer/exporter/tools/adapter/net_dict.go

```diff
@@ -18,6 +18,7 @@ const (
 	GRPC
 	DUBBO
 	REDIS
+  ROCKETMQ
```

Comment:
We use tabs for indentation and gofmt emits them by default.Use spaces only if you must.

Reply:
> We use tabs for indentation and gofmt emits them by default.Use spaces only if you must.

I thought I'd dealt with this place, but I haven't,I will use gofmt now